pkgs/development/python-modules/whirlpool-sixth-sense/default.nix

```diff
@@ -12,7 +12,7 @@
 
 buildPythonPackage rec {
   pname = "whirlpool-sixth-sense";
-  version = "unstable-2021-08-22";
+  version = "unstable-0.15.1";
```

Comment:
This doesn't look correct.

Reply:
Indeed, that is an unintended downgrade gone wrong. Dropped the commit.